internal/testutils/database.go

```diff
@@ -147,3 +151,17 @@ func testDbString(dbname string) string {
 		config.Get().DatabasePort,
 	)
 }
+
+func ConnectAndMigrateDB(packageName string) {
+	ConnectToTestDB(packageName)
+
+	if out := dao.DB.Exec("CREATE SCHEMA " + packageName); out.Error != nil {
+		fmt.Println("error in creating schema " + out.Error.Error())
+	}
+
+	MigrateSchema()
+
+	if out := dao.DB.Exec("SET search_path TO \"$user\", " + packageName); out.Error != nil {
+		fmt.Println("error in setting schema" + out.Error.Error())
+	}
```

Comment:
I would suggest using [`log.Fatalf`](https://pkg.go.dev/log#Fatalf) here, as I guess it makes sense just to end the program if the schema creation or the migrations don't work:

```go
import "log"

//...

if out := dao.DB.Exec("CREATE SCHEMA " + packageName); out.Error != nil {
	log.Fatalf("error in creating schema: %s", out.Error)
}

MigrateSchema()

if out := dao.DB.Exec("SET search_path TO \"$user\", " + packageName); out.Error != nil {
	log.Fatalf("error in setting schema: %s", out.Error)
}
```